lefthook.yml

```
@@ -0,0 +1,18 @@
+# lefthook.yml - Git hooks configuration
+# https://lefthook.dev/configuration/
+
+pre-commit:
+  parallel: true
+  commands:
+    lint:
+      glob: "*.{js,ts,tsx}"
+      run: bun run lint
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)